BitwardenShared/UI/Vault/VaultItem/ViewItem/ViewIdentityItem/ViewIdentityItemView.swift

```diff
@@ -12,82 +12,109 @@ struct ViewIdentityItemView: View {
             BitwardenTextValueField(
                 title: Localizations.company,
-                value: store.state.company,
-                valueAccessibilityIdentifier: "IdentityCompanyEntry"
+                value: company,
+                valueAccessibilityIdentifier: "IdentityCompanyEntry",
+                copyButtonAction: { store.send(.copyPressed(value: company, field: .company)) },
+                copyButtonAccessibilityIdentifier: "IdentityCopyCompanyButton"
             )
             .accessibilityElement(children: .contain)
         }
 
         if !store.state.socialSecurityNumber.isEmpty {
+            let socialSecurityNumber = store.state.socialSecurityNumber
             BitwardenTextValueField(
                 title: Localizations.ssn,
-                value: store.state.socialSecurityNumber,
-                valueAccessibilityIdentifier: "IdentitySsnEntry"
+                value: socialSecurityNumber,
+                valueAccessibilityIdentifier: "IdentitySsnEntry",
+                copyButtonAction: { store.send(.copyPressed(value: socialSecurityNumber, field: .socialSecurityNumber)) },
+                copyButtonAccessibilityIdentifier: "IdentityCopySocialSecurityNumberButton"
             )
             .accessibilityElement(children: .contain)
         }
 
         if !store.state.passportNumber.isEmpty {
+            let passportNumber = store.state.passportNumber
             BitwardenTextValueField(
                 title: Localizations.passportNumber,
-                value: store.state.passportNumber,
-                valueAccessibilityIdentifier: "IdentityPassportNumberEntry"
+                value: passportNumber,
+                valueAccessibilityIdentifier: "IdentityPassportNumberEntry",
+                copyButtonAction: { store.send(.copyPressed(value: passportNumber, field: .passportNumber)) },
+                copyButtonAccessibilityIdentifier: "IdentityCopyPassportNumberButton"
             )
             .accessibilityElement(children: .contain)
         }
 
         if !store.state.licenseNumber.isEmpty {
+            let licenseNumber = store.state.licenseNumber
             BitwardenTextValueField(
                 title: Localizations.licenseNumber,
-                value: store.state.licenseNumber,
-                valueAccessibilityIdentifier: "IdentityLicenseNumberEntry"
+                value: licenseNumber,
+                valueAccessibilityIdentifier: "IdentityLicenseNumberEntry",
+                copyButtonAction: { store.send(.copyPressed(value: licenseNumber, field: .licenseNumber)) },
+                copyButtonAccessibilityIdentifier: "IdentityCopyLicenseNumberButton"
             )
             .accessibilityElement(children: .contain)
         }
 
         if !store.state.email.isEmpty {
+            let email = store.state.email
             BitwardenTextValueField(
-                title: Localizations.email,
+                title: email,
```

Comment:
⚠️ I believe the title should be the localized string `email`. Was there a product change for this?